test/parallel/test-next-tick-intentional-starvation.js

```diff
@@ -23,12 +23,6 @@
 require('../common');
 const assert = require('assert');
 
-// this is the inverse of test-next-tick-starvation.
```

Comment:
Could we keep the test description (these 3 lines)

Reply:
I'll revert except the third line